app/scripts/controllers/replicaSet.js

```diff
@@ -0,0 +1,448 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:ReplicaSetController
+ * @description
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('ReplicaSetController',
+              function ($scope,
+                        $filter,
+                        $routeParams,
+                        AlertMessageService,
+                        DataService,
+                        HPAService,
+                        MetricsService,
+                        ProjectsService,
+                        DeploymentsService,
+                        ImageStreamResolver,
+                        Navigate,
+                        keyValueEditorUtils,
+                        kind) {
+    var hasDC = false;
+    var displayKind = $filter('humanizeKind')(kind);
+    switch (kind) {
+    case 'ReplicaSet':
+      $scope.breadcrumbs = [
+        {
+          title: "Replica Sets",
```

Comment:
@jwforres forgot to make this change to all of the edit pages. might want a utility that builds breadcrumbs from the resource?